react-app/src/components/ProposalDetailScreen/ProposalDetailScreen.tsx

```diff
@@ -1,21 +1,41 @@
 import React from "react";
 import { Navigate, useParams } from "react-router-dom";
-import cn from "classnames";
+import {
+  Proposal,
+  ProposalStatus,
+  ProposalType,
+} from "../../generated/graphql";
 import AppRoutes from "../../navigation/AppRoutes";
 import Paper from "../common/Paper/Paper";
+import ProposalHeader from "./ProposalHeader";
 
 const ProposalDetailScreen: React.FC = () => {
   const { id } = useParams();
 
   if (!id) return <Navigate to={AppRoutes.Proposals} />;
 
+  // mock data, remove after API integration
+  const proposal: Proposal = {
+    id,
+    description: "description",
+    type: ProposalType.Text,
+    proposerAddress: "like1ewpwcdfgsdfdu0jj2unwhjjl58yshm9xnvr9c2",
+    status: ProposalStatus.VotingPeriod,
+    title:
+      "Community-pool-spend Proposal: deposit 2,000,000 LIKE to Tech Subdao multisig wallet",
+    depositEndTime: new Date(),
+    submitTime: new Date(),
+    votingEndTime: new Date(),
+    votingStartTime: new Date(),
+  };
+
   return (
-    <div className={cn("flex", "flex-col")}>
-      <Paper>#{id} Proposal Summary Placeholder</Paper>
+    <>
```

Comment:
Root element of a screen component should be wrapped with a concrete element like `div`. Otherwise the layout may be affected by the one render the component